src/drivers/imx/interrupt.c

```diff
@@ -363,6 +363,20 @@ static const struct irq_cascade_tmpl dsp_irq[] = {
 	IRQSTR_CASCADE_TMPL_DECL(7)
 };
 
+int irqstr_get_sof_int(int irqstr_int)
+{
+	int line, irq;
+
+	/* Is it a valid interrupt? */
+	if (irqstr_int < 0 || irqstr_int >= 512)
```

Comment:
Maybe you should replace 512 with some platform define.
It will be more flexible in case there will be any other platform flavor in the future.
Also it's more descriptive.

Reply:
Reusing IRQSTR_IRQS_NUM for this one. Forgot I put it in there.